src/components/NetworkDropdown/NetworkDropdown.tsx

```diff
@@ -116,6 +126,34 @@ function DesktopDropdown({ setActiveModal, selectorLabel, networkOptions, onNetw
             />
           </div>
           <div className="network-dropdown-divider" />
+          {/* <Menu.Item>
+            <div className="network-dropdown-menu-item menu-item" onClick={setApprovalsModalVisible}>
+              <div className="menu-item-group">
+                <div className="menu-item-icon">
+                  <img className="network-dropdown-icon" src={clickTradingIcon} alt="" />
+                </div>
+                <span className="network-dropdown-item-label">
+                  <Trans>{`Enable One-Click Trading`}</Trans>
+                </span>
+              </div>
+            </div>
+          </Menu.Item> */}
+          {/* <Menu.Item>
+            <div
+              className="network-dropdown-menu-item menu-item last-dropdown-menu"
+              onClick={() => setActiveModal(LANGUAGE_MODAL_KEY)}
+            >
+              <div className="menu-item-group">
+                <div className="menu-item-icon">
+                  <img className="network-dropdown-icon" src={emailIcon} alt="" />
+                </div>
+                <span className="network-dropdown-item-label">
+                  <Trans>{`Enable Email Notifications`}</Trans>
+                </span>
+              </div>
+            </div>
+          </Menu.Item>
+          <div className="network-dropdown-divider" /> */}
```

Comment:
Do we intend to eventually re-enable this?

Reply:
Yes, I'll work on this in the subsequent PR.